### DISCUSSION TITLE
App available on FDroid official repo?

### DISCUSSION THREAD

**Discussion by Kazing** [CappielloAntonio/tempo#101] [Q&A] [answered]
Hi,
I use this app and really like it.
Will it be available on FDroid? I saw that it was discussed, what's the status?

Thanks

**Comment by rkowalsk** [comment]
It is available on the IzzyOnDroid f-droid repo.

**Reply by Kazing** [reply]
Thanks, but I was interested by the official FDroid repo.

@CappielloAntonio : I found that there is an request opened for Tempo, what is the status ? : https://gitlab.com/fdroid/fdroiddata/-/merge_requests/13554

**Reply by inson1** [reply, marked as answer]
@Kazing now its merged (another mr) and it is there :) time to close this question

**Reply by Kazing** [reply]
Thanks i am closing this !
